software/firmware/europi.py

```diff
@@ -326,7 +380,13 @@ def __init__(self, sda, scl, width=OLED_WIDTH, height=OLED_HEIGHT, channel=I2C_C
         super().__init__(self.width, self.height, i2c)
 
     def clear(self):
-        """Clear the display upon call."""
+        """Clear the display upon call.
+
+        Note, this is meant to be a standalone call to clear the screen because
+        it also calls ``show()``. Using this in a loop with other oled 
+        functions will cause the screen to flicker. Instead you want to start
+        your loops with ``oled.fill(0)`` to clear the FrameBuffer.

```

Comment:
I wonder if we should just remove this method entirely. It seems to cause more confusion than utility. (It's not used in any contrib scripts)

Reply:
I think (at least personally) it's a confusing/unusable function because it contains the .show(). If it was simply to reword .fill(0) to .clear() then it would be fine, and that rewording makes sense for intuitition's sake, but including the .show() means you can't use it in any case where you're relying on only a single .show() at end for speed

Reply:
Opened this as issue #97